SECURITY.md

```diff
@@ -4,18 +4,18 @@
 
 At Autodesk, we know that the security of your data is critical to your studio’s
 operation.
-As the industry shifts to the cloud, ShotGrid knows that security and service
+As the industry shifts to the cloud, Flow Production Tracking knows that security and service
```

Comment:
The new name being larger, you need to check the line size rule in this document